src/components/formSections/components/TermsListField.js

```diff
@@ -246,29 +252,49 @@ export default class TermsListField extends React.Component {
     );
   }
 
+  renderTermsField = () => {
+    let termNoteError = false;
+    const { terms } = this.state;
+    return terms.map((term, i) => {
```

Comment:
Can you clean up the logic of this function a bit? Rather than `return`ing here, just save the mapped array to a variable. Then you don't need to do the weird `if` block at L267 and can just check if there's an error after the map().

Also, I think the whole `err`, `errorMessage` and `termNoteError` usage is kind of overblown. If you do the above suggestion, you only really need to define one `let` variable at the top of this function and use it for everything, assuming you return the actual error message from within `validateNoteField`.

This whole thing is super close but let's please clean it all up and make it easier to reason through.